global-hooks/openapi/values.yaml

```diff
@@ -1,105 +1,347 @@
-############
-# Attention!!! This schema exists only for matrix testing purposes.
-# Please, consider it completely rewritable.
-############
-
+x-extend:
+  schema: config-values.yaml
 type: object
 default: {}
-additionalProperties: true
+additionalProperties: false
 properties:
-  deckhouseVersion:
-    type: string
-    x-examples: ["dev"]
-  clusterConfiguration:
+  internal:
+    additionalProperties: false
+    type: object
     default: {}
-    additionalProperties: true
-    x-examples:
-    - podSubnetNodeCIDRPrefix: "24"
-      podSubnetCIDR: 10.244.0.0/16
-      serviceSubnetCIDR: 192.168.0.0/16
-      kubernetesVersion: "1.19"
-      clusterDomain: test
-      clusterType: "Cloud"
-      cloud:
-        prefix: test
-        provider: OpenStack
-  highAvailability:
+    properties:
+      modules:
+        default: { }
+        additionalProperties: false
+        type: object
+        properties:
+          resourcesRequests:
+            type: object
+            default: {}
+            additionalProperties: false
+            properties:
+              milliCpuEveryNode:
+                type: number
+                format: double
+                minimum: 0
+              memoryEveryNode:
+                type: integer
+                format: int64
+                minimum: 0
+              milliCpuControlPlane:
+                type: integer
+                format: int64
+                minimum: 0
+              memoryControlPlane:
+                type: integer
+                format: int64
+                minimum: 0
+              milliCpuMaster:
+                type: integer
+                format: int64
+                minimum: 0
+              memoryMaster:
+                type: integer
+                format: int64
+                minimum: 0
+  clusterConfiguration:
+    # TODO need to use relative path instead absolute
```

Comment:
We are ok with using the `/deckhouse` path for now.

Reply:
I removed comment